two_factor/views/core.py

```diff
@@ -530,9 +530,12 @@ def get_device(self, **kwargs):
 
     def get_key(self, step):
         self.storage.extra_data.setdefault('keys', {})
+
         if step in self.storage.extra_data['keys']:
             return self.storage.extra_data['keys'].get(step)
+
         key = random_hex_str(20)
+
```

Comment:
Why all these random newlines? I don't see how they are related to your proposed change.